src/app/contact/page.tsx

```diff
@@ -1,53 +1,29 @@
 import Typography from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import { Metadata } from 'next';
-import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
 
-import ExternalLink from '@/components/ExternalLink';
 import PageContainer from '@/components/PageContainer';
+import ExternalLink from '@/components/ExternalLink';
 
 export const metadata: Metadata = {
   title: 'Contact',
 };
 
 const Contact: React.FC = () => {
   return (
-    <PageContainer>
+    <PageContainer data-testid="contact-page">
       <Box component="section">
-        <Typography variant="h1">Contact</Typography>
+        <Typography variant="h1" gutterBottom>
+          Contact
+        </Typography>
         <Typography>
-          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ipsum
-          tempore a praesentium perferendis doloremque, veniam minus quis
-          laborum, numquam blanditiis sunt ex consectetur asperiores assumenda
-          nisi laboriosam et tempora quos.
+          Please connect with me on{' '}
+          <ExternalLink href="https://www.linkedin.com/in/nicholeuf">
```

Comment:
I would like to fill this page out more but am not feeling creative with this right now. I will think about ideas and open a new PR later.